docs/local_dev.md

```diff
@@ -179,3 +179,115 @@ kubectl delete cluster $CLUSTER_NAME
 docker rm -f $HOST_NAME
 kind delete cluster
 ```
+
+# Host Agent Installer
+The installer is responsible for detecting the BYOH OS, downloading a BYOH bundle and installing/uninstalling it.
+
+## Supported OS and Kubernetes
+The current list of supported tuples of OS, kubernetes Version, BYOH Bundle Name can be retrieved with:
+```shell
+./cli --list-supported
+```
+An example output looks like:
+<table>
+    <tr>
+        <td>OS</td>
+        <td>K8S Version</td>
+        <td>BYOH Bundle Name</td>
+    </tr>
+    <tr>
+        <td>Ubuntu_20.04.*_x86-64</td>
+        <td>v1.22.3</td>
+        <td>byoh-bundle-ubuntu_20.04.1_x86-64_k8s_v1.22.3</td>
+    </tr>
+</table>
+The '*' in OS means that all Ubuntu 20.04 patches will be handled by this BYOH bundle.
+
+## Pre-requisites
+As of writing this, the following packages must be pre-installed on the BYOH host:
```

Comment:
Question: These packages are only required for debian systems or other systems as well? Do we need to mention Ubuntu specifically?
```suggestion
As of writing this, the following packages must be pre-installed on the Host:
```

Reply:
As of now we support only Ubuntu. For RHEL these will be rpms